gzip.go

```diff
@@ -1,13 +1,13 @@
-package main
+package tileutils
```

Comment:
technically this package should be called `baremaps-exporter`. If we are going to call this `tileutils` we should probably put it in a `pkg/tileutils` directory to follow golang name standards.

Reply:
I wanted a one word name, so I moved it to `pkg/tileutils`.